trunk/NDHMS/Routing/module_RT.F

```diff
@@ -350,7 +352,6 @@ subroutine rt_allocate(did,ix,jx,ixrt,jxrt,nsoil,CHANRTSWCRT)
 
         allocate ( rt_domain(did)%reservoirs(NLAKES) )    ! allocate array of pointers to reservoirs
         allocate ( rt_domain(did)%q_type(NLAKES) )        ! allocate array to specify type of reservoir
```

Comment:
https://github.com/NCAR/wrf_hydro_nwm_public/pull/203/files#diff-47462492064b8fa54cb26c3623ef1dcdR5
This `rt_domain(did)%q_type` variable snuck in in PR #203  which was previous work on levelpool. 
I really dislike this variable name and I think we should fix it in this PR #259 . 
The variable is way too vague to be in `rt_domain`. It should be `reservoir_type` or something with reservoir in the name (`reservoir_q_type`?).

Reply:
In the comment and metadata, this variable is referred to as "reservoir_type"
I will apply this change. It's not heinous.
@jmccreight

Reply:
FIxed in b2133f8